ArduCopter/commands_logic.cpp

```diff
@@ -1132,16 +1131,13 @@ void Copter::do_digicam_control(const AP_Mission::Mission_Command& cmd)
                        cmd.content.digicam_control.cmd_id)) {
         log_picture();
     }
-#endif
 }
 
 // do_take_picture - take a picture with the camera library
 void Copter::do_take_picture()
```

Comment:
The declaration of this one isn't surrounded by the CAMERA ifdef, have you tested this?